package.json

```diff
@@ -36,13 +36,14 @@
     "dependencies": {
         "@babel/runtime": "^7.19.4",
         "crypto-js": "3.3.0",
-        "expo-constants": "^14.2.1",
+        "expo-constants": "^15.4.5",
+        "expo-manifests": "^0.13.2",
         "expo-modules-core": "^1.2.6",
-        "expo-web-browser": "12.1.1",
-        "expo-secure-store": "^12.1.1",
+        "expo-secure-store": "*",
+        "expo-web-browser": "*",
```

Comment:
I don't believe we should be using `*` here

Reply:
Didn't put that there